develop/dev-guide-proxysql-integration.md

```diff
@@ -104,579 +655,785 @@ The production environment requires more steps than the test environment. To dep
+> **Note:**
+>
+> The following sections list only the required configuration items.
+> For a comprehensive list of configurations, see [ProxySQL documentation](https://proxysql.com/documentation/proxysql-configuration/).
 
-### Simple introduction
+##### Option 1: Configuring ProxySQL using the Admin Interface
 
-ProxySQL uses a port to manage configuration, which is **_ProxySQL Admin interface_**, and a port to proxy, which is **_ProxySQL MySQL Interface_**.
+1. Reconfigure ProxySQL’s internals using the standard ProxySQL the Admin interface, accessible via any MySQL command line client (available by default on port `6032`):
 
-- **_ProxySQL Admin interface_**: To connect to the admin interface, you can use an `admin` user to read and write configuration, or use a `stats` user to read part of statistics (cannot read or write configuration). The default credentials are `admin:admin` and `stats:stats`. For security reasons, you can use the default credentials to connect locally, but to connect remotely, you need to configure a new user, which is often named `radmin`.
-- **_ProxySQL MySQL Interface_**: Used as a proxy to forward SQL to the configured service.
+    ```bash
+    mysql -u admin -padmin -h 127.0.0.1 -P6032 --prompt 'ProxySQL Admin> '
+    ```
 
-![proxysql config flow](/media/develop/proxysql_config_flow.png)
+    The above step will take you to the ProxySQL admin prompt.
 
-There are three layers in ProxySQL configurations: `runtime`, `memory`, and `disk`. You can change the configuration of the `memory` layer only. After modifying the configuration, you can use `LOAD xxx TO runtime` to make the configuration effective, and/or you can use `SAVE xxx TO DISK` to save to the disk to prevent configuration loss.
+2. Configure the TiDB servers to be used, where you can add one or multiple TiDB servers to ProxySQL. In this example, one TiDB server is added as follows:
 
-![proxysql config layer](/media/develop/proxysql_config_layer.png)
+    ```sql
+    INSERT INTO mysql_servers(hostgroup_id, hostname, port) VALUES (0, '<tidb cloud dedicated cluster host>', <tidb cloud dedicated cluster port>);
+    LOAD mysql servers TO runtime;
+    SAVE mysql servers TO DISK;
+    ```
 
-### Configure TiDB server
+    > **Note:**
+    >
+    > `hostgroup_id`: ProxySQL manages servers using hostgroup. To distribute SQL traffic to these servers evenly, you can configure several servers that need load balancing to the same hostgroup. To distinguish the servers, such as for read and write purposes, you can configure them to use different hostgroup.
+    > `hostname`: The IP address or domain of the TiDB server.
+    > `port`:  The port of TiDB server.
 
-You can add multiple TiDB servers in ProxySQL. To add TiDB servers, perform the following at **_ProxySQL Admin interface_**:
+3. Configure Proxy login users to make sure that the users have appropriate permissions on the TiDB server:
```

Comment:
@Icemap change
 _Configure Proxy login users to make sure that the users have appropriate permissions on the TiDB server_
**WITH**
Configure Proxy login users to make sure that the users have appropriate permissions on the TiDB server. In the code below  Replace **'tidb cloud dedicated cluster username'** and **'tidb cloud dedicated cluster password'** with the cluster `hostname` and `root password` that's been configured.